src/auth/services/team.service.ts

```
@@ -22,6 +22,8 @@ import type {
   GetTeamsDto,
 } from '../dto/manage-team.dto';
 import { UserRole } from '../enums/userRole.enum';
+import { TeamMember } from '../entities/team-member.entity';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'TeamMember'. Consider to delete it or rename it.

'TeamMember' is defined here:


(lint/suspicious/noRedeclare)